packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -1440,24 +1422,38 @@ class CollectSameUntilThresholdRound(CollectionRound, ABC):
     none_event: Any
     collection_key: str
     selection_key: str
+    save_all_values: bool = False
```

Comment:
For now this approach is taken to phase in the new tuple approach. It's a bit hacky but reduces changes required on skills.